### ISSUE TITLE
Tesseract OCR Support Not Working

### ISSUE THREAD

**Issue by HBKarlHolzinger** [dadoonet/fscrawler#444]
FS Crawler version: 2.4
Tesseract-OCR version: 3.05
Elasticsearch version: 5.6.1
Windows Environment

Following the directions in the readme, I have installed Tesseract for OCR support. However, FS Crawler does not seem to recognize the installation. I've attached the output from my attempt, and these 2 lines express this problem:

```
11:25:08,631 [36mDEBUG[m [f.p.e.c.f.t.TikaInstance] OCR is activated for PDF documents
11:25:08,641 [36mDEBUG[m [f.p.e.c.f.t.TikaInstance] But Tesseract is not installed so we won't run OCR.
```

I can send commands directly to Tesseract and get the expected results, but FS Crawler does not seem to recognize that Tesseract is installed. The path to my Tesseract installation is C:\Program Files (x86)\Tesseract-OCR\tesseract.exe

[ocr_test_log.txt](https://github.com/dadoonet/fscrawler/files/1337957/ocr_test_log.txt)

**Comment by kinbod** [comment]
yes, I also meet this problem.

**Comment by dadoonet** [comment]
I recently merged #520 which I hope could fix the problem in case Tesseract is not correctly detected.
Could you try the latest snapshot and report in #520 if this is fixed?
